### ISSUE TITLE
boolean property with default: false became required

### ISSUE THREAD

**Issue by Ti-webdev** [ferdikoomen/openapi-typescript-codegen#636]
My schema is:
```yaml
components:
  schemas:
    AuthRequest:
      type: object
      properties:
        email:
          type: string
          format: email
        password:
          type: string
          format: byte
          description: base64-encoded password
        remember:
          type: boolean
          default: false
      required:
        - email
        - password
```

it's generated to:
```typescript
$AuthRequest = {
    properties: {
        email: {
            type: 'string',
            isRequired: true,
            format: 'email',
        },
        password: {
            type: 'string',
            isRequired: true,
            format: 'byte',
        },
        remember: {
            type: 'boolean',
            isRequired: true,
        },
    },
};
```

`remember` is marked as `isRequired: true` but it isn't required.

**Comment by Ti-webdev** [comment]
May be the logic should be inverted?

https://github.com/ferdikoomen/openapi-typescript-codegen/blob/830b317d0a0f6aaa97dba43818ce6d1ccc109be2/src/openApi/v3/parser/getModel.ts#L26

**Comment by ibrahimduran** [comment]
I think that is because TypeScript syntax doesn't allow optional params with default value. So fields with default value are basically treated as required while default value is added in code, which makes them optional.

Problem is default values are not set for `requestBody` in [getOperationRequestBody](https://github.com/ferdikoomen/openapi-typescript-codegen/blob/715ddce21a3dc198fdd0f3a70f60da0503f22d7c/src/openApi/v3/parser/getOperationRequestBody.ts#L39), which may be related to the #257 bu I'm not sure.https://github.com/ferdikoomen/openapi-typescript-codegen/blob/715ddce21a3dc198fdd0f3a70f60da0503f22d7c/src/openApi/v3/parser/getOperation.ts#L61-L69

**Comment by ferdikoomen** [comment]
Hi @Ti-webdev this issue should be fixed on 0.11.x
